src/views/donationView/index.tsx

```diff
@@ -96,6 +107,76 @@ export default function DonationView({ donations }: DonationViewProps) {
     },
   ];
 
+  // Filter columns based on visibleColumns selection
+  const columns = allColumns.filter((column) =>
+    visibleColumns.includes(column.field)
+  );
+
+  const handleColumnSelectionChange = (event: SelectChangeEvent<string[]>) => {
+    const selectedColumns = event.target.value as string[];
+    setVisibleColumns(selectedColumns);
+  };
+
+  const CustomToolbar = () => (
+    <Box sx={{ p: 1, display: 'flex', justifyContent: 'space-between' }}>
+      <Box sx={{ display: 'flex', gap: 1 }}>
+        <FormControl sx={{ minWidth: 120 }}>
+          <InputLabel></InputLabel>
+          <Select
+            multiple
+            value={visibleColumns}
+            onChange={handleColumnSelectionChange}
+            renderValue={(selected) =>
+              selected.length === 0 ? 'Select Columns' : 'Columns'
+            }
+          >
+            {allColumns.map((column) => (
+              <MenuItem key={column.field} value={column.field}>
+                <Checkbox checked={visibleColumns.includes(column.field)} />
+                <ListItemText primary={column.headerName} />
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+        <Button variant="contained" color="primary" onClick={exportToCSV}>
```

Comment:
Can we use the outlined variant here? It's more of a secondary action if that makes sense

Reply:
Also, if possible, reduce the size of the button, it looks a bit too tall. OK if this is not feasible, let me know either way

Reply:
I have made the changes for 
-  prevent the rows from being clickable
- csv does not contain a column for the edit icon
- outlined variant
- reduce button size